app/routes/wow.ilvl-export-search.tsx

```
@@ -0,0 +1,506 @@
+          name="sortBy"
+          value={formValues.sortBy}
+          options={[
+            { label: 'Minimum Price', value: 'minPrice' },
+            { label: 'Item Quantity', value: 'itemQuantity' },
+            { label: 'Realm Population', value: 'realmPopulationReal' },
+            { label: 'Realm Ranking', value: 'realmRanking' }
+          ]}
+          onChange={(e) => setFormValues(prev => ({ ...prev, sortBy: e.target.value }))}
+        />
+        <div className="flex flex-col gap-2">
+          <label className="text-sm font-medium dark:text-gray-200">Desired Stats</label>
```

Biome (1.9.4)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)